example/lib/widgets/notice_banner.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+enum NoticeBannerType {
```

Comment:
If we use named constructors for the `NoticeBanner` we don't need an extra enum.

Reply:
Good idea, I'll do that.

Reply:
@josxha Completed in 40a5019.